app/services/search/query_builders/feed_content.rb

```diff
@@ -64,10 +69,17 @@ def initialize(params)
         @params[:published] = true
 
         build_body
+        add_function_scoring unless sort_params_present?
```

Comment:
What if we moved this inside and at the end of `build_body` since `add_function_scoring` depends on `build_body` being executed first and it reassigns the value of `@body[:query]`? It feels a tad more abstract that way, but there's less of a chance that in a future PR this order is changed up by someone else by mistake and it still fits within the concept of building the query body.

Reply:
Definitely thought about that but didnt want to override the whole build body method. I was also hesitant to add another empty method to query_base.rb like I have been doing
```ruby
def add_function_scoring; end
```

However, in the future, if we want to use this on other models having it there would be nice.